test/Compiler-caching.test.js

```diff
@@ -14,7 +15,7 @@ describe("Compiler (caching)", function() {
 		new WebpackOptionsDefaulter().process(options);
 		options.entry = entry;
 		options.context = path.join(__dirname, "fixtures");
-		options.output.path = "";
+		options.output.path = __dirname;
```

Comment:
This should be `"/"`.

Set `c.outputFileSystem.join` to `function() { return [].join.call(arguments, "/").replace(/\/+/g, "/"); }` this should only add `/` as separator.